src/app/profile/addActivityModal/AddActivityModal.tsx

```diff
@@ -1,20 +1,28 @@
 import { Box, Modal, Typography } from '@mui/material';
+import { focusManager } from 'react-query';
 
-import { useLocale } from 'hooks';
+import { useLocale, useSnackbar } from 'hooks';
 import { styles } from './AddActivityModal.styles';
 import { AddActivityModalProps } from './AddActivityModal.types';
-import { AddActivityForm } from './addActivityForm/AddActivityForm';
+import { AddActivityFormContainer } from './addActivityForm/AddActivityFormContainer';
 
-export const AddActivityModal = ({ isOpened, onModalClose }: AddActivityModalProps) => {
+export const AddActivityModal = ({ isOpened, onModalClose, name }: AddActivityModalProps) => {
   const { formatMessage } = useLocale();
+  const { showSnackbar } = useSnackbar();
+
+  const onSubmit = () => {
+    showSnackbar(formatMessage({ id: 'addActivity.success' }));
+    onModalClose();
+    focusManager.setFocused(true);
+  };
 
   return (
     <Modal open={isOpened} onClose={onModalClose} aria-labelledby="modal-add-activity">
       <Box sx={styles.modal}>
         <Typography variant="h6" sx={styles.header}>
           {formatMessage({ id: 'profile.addActivity' })}
         </Typography>
-        <AddActivityForm onSubmit={(data) => console.log(data)} />
+        <AddActivityFormContainer name={name} onFormSubmission={onSubmit} />
```

Comment:
Jak dobrze rozumiem, to co wchodzi do `onFormSubmission` wywołuje się wtedy, kiedy dane się zapisują poprawnie, prawda? Więc lepiej byłoby to nazwać `onSave`